values/knative-operator/knative-operator.gotmpl

```diff
@@ -4,20 +4,18 @@
 knative_operator:
   knative_operator:
     resources: {{- $k.resources.operator | toYaml | nindent 6 }}
-    image:
-      {{- if $v.otomi.linodeLkeImageRepository }}
-      repo: "{{- $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/operator/cmd/operator"
-      {{- else }}
-      repo: gcr.io/knative-releases/knative.dev/operator/cmd/operator
-      {{- end }}
-      tag: v1.15.6
+    {{- if $v.otomi.linodeLkeImageRepository }}
+    image: "{{- $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/operator/cmd/operator"
+    {{- else }}
+    image: gcr.io/knative-releases/knative.dev/operator/cmd/operator
+    {{- end }}
+    tag: v1.15.6
 
   operator_webhook:
     resources: {{- $k.resources.operatorWebhook | toYaml | nindent 6 }}
-    image:
-      {{- if $v.otomi.linodeLkeImageRepository }}
-      repo: "{{- $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/operator/cmd/webhook"
-      {{- else }}
-      repo: gcr.io/knative-releases/knative.dev/operator/cmd/webhook
-      {{- end }}
-      tag: v1.15.6
\ No newline at end of file
+    {{- if $v.otomi.linodeLkeImageRepository }}
+    image: "{{- $v.otomi.linodeLkeImageRepository }}/gcr/knative-releases/knative.dev/operator/cmd/webhook"
+    {{- else }}
+    image: gcr.io/knative-releases/knative.dev/operator/cmd/webhook
+    {{- end }}
+    tag: v1.15.6
```

Comment:
I see that the `tag` was previously not effective, since it was in the wrong location. But I am also wondering why I left it in there in the first place instead of using the Helm chart default (v1.18.x). Is there a reason to downgrade now?